### ISSUE TITLE
Appgate Site VPN Route_Via type Error

### ISSUE THREAD

**Issue by infosec-shinobi** [appgate/terraform-provider-appgatesdp#203]
### Terraform CLI, Terraform appgatesdp Provider Version, and Appgate Appliance version(s)

terraform cli: v0.13.7
terraform-provider-appgatesdp:  v0.8.3
appgate appliance version: 5.3.3

### Affected Resource(s)

* appgatesdp_site

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "appgatesdp_site" "default_test_site" {
    name = "default test"
    short_name = "DT"
    entitlement_based_routing = false
    notes = "Managed by terraform"
    default_gateway {
        enabled_v4       = true
        enabled_v6       = false
    }
    name_resolution {
        use_hosts_file = false
        dns_resolvers {
            name = "test"
            update_interval = 60
            servers = ["10.10.10.10"]
            search_domains = ["example.com"]
        }
    }
    vpn {
        state_sharing = false
        ip_access_log_interval_seconds = 120
        snat = false
        tls {
          enabled = true
        }
        dtls {
            enabled = false
        }
        route_via {
            ipv4 = "10.10.10.10"
            ipv6 = "fdf8:f53b:82e4::53"
        }
      }
    tags = ["terraform", "api-created","default_test_site"]
}
```

### Debug Output

N/A

### Panic Output

```
panic: interface conversion: interface {} is map[string]interface {}, not *schema.Set
...
2021-12-27T15:19:34.937Z [DEBUG] plugin.terraform-provider-appgatesdp_v0.8.3: github.com/appgate/terraform-provider-appgatesdp/appgate.resourceAppgateSiteRead()
2021-12-27T15:19:34.937Z [DEBUG] plugin.terraform-provider-appgatesdp_v0.8.3: 	github.com/appgate/terraform-provider-appgatesdp/appgate/resource_appgate_site.go:601
...
2021-12-27T15:19:34.941Z [DEBUG] plugin: plugin process exited: path=.terraform/plugins/registry.terraform.io/appgate/appgatesdp/0.8.3/linux_amd64/terraform-provider-appgatesdp_v0.8.3 pid=320 error="exit status 2"
2021-12-27T15:19:34.941Z [WARN]  plugin.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2021/12/27 15:19:34 [DEBUG] appgatesdp_site.dv_test_site: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2021/12/27 15:19:34 [TRACE] eval: *terraform.EvalMaybeTainted
2021/12/27 15:19:34 [TRACE] EvalMaybeTainted: appgatesdp_site.dv_test_site encountered an error during creation, so it is now marked as tainted
...
```

### Expected Behavior

Site is created/read/updated

### Actual Behavior

Appgate throws RPC errors and then the rest of the plan/apply bomb.

### Steps to Reproduce

1. Try to create/manage a site that has values set for "route_via" under vpn section.

### Important Factoids

N/A

### References

N/A

I believe the issue is that route_via is being returned as the wrong type causing the provider to crash.

**Comment by infosec-shinobi** [comment]
I think this should be addressed in my latest commit (67d76c7749e79fe1c1826e9ab269bb9474ca3c8b) to #201

**Comment by danijeel** [comment]
Closing this issue, this should be resolved in [v0.8.4](https://github.com/appgate/terraform-provider-appgatesdp/releases/tag/v0.8.4)
